docs/website/views/_layouts/popular-articles.njk

```diff
@@ -0,0 +1,17 @@
+---
+layout: content
+---
+
+<h2 class="popular-articles-title">Welcome on Exadel authoring Kit for AEM</h2>
+<h3 class="popular-articles-subheader">Popular articles</h3>
+<div>
```

Comment:
cosmetic: we can remove this extra div, since it isn't needed here, it seems to me